nilearn/image/image.py

```diff
@@ -60,15 +60,15 @@ def high_variance_confounds(imgs, n_confounds=5, percentile=2.,
             If provided, confounds are extracted from voxels inside the mask.
             See http://nilearn.github.io/manipulating_images/input_output.html.
 
-        n_confounds : :obj:`int`
+        n_confounds : int
```

Comment:
why is this file modified?

Reply:
Looking at the file history it seems I forgot spaces I added after parameter names to comply with the NumPy docstring formatting. I'll revert it to c10248e43769f37eaea804f64d44a7816e3c6e03.